src/main/java/nurseybook/model/NurseyBook.java

```diff
@@ -191,6 +191,12 @@ public boolean hasTask(Task t) {
         return tasks.contains(t);
     }
 
+    @Override
+    public ObservableList<Task> getRealTaskList() {
+        UniqueTaskList realTaskList = tasks.getRealTaskList();
```

Comment:
Is the rationale for this method to ensure that only real tasks are saved? Also, can I check whether the list returned is intended?  `tasks.getRealTaskList()` returns a new UniqueTaskList, but the method just returns the `internalUnmodifiableList` of the original tasklist?